modules/wizard/view-wizard-step-home-page.php

```diff
@@ -63,7 +69,10 @@
 	<tbody>
 	<?php
 	foreach ( array_keys( $translations ) as $lang ) {
-		$language = $this->model->get_language( $lang );
+		$language = $model->languages->get( $lang );
+		if ( empty( $language ) ) {
+			continue;
+		}
```

Comment:
Is it possible to have an empty language here? If not just document:
```suggestion
		/** @var PLL_language */
		$language = $model->languages->get( $lang );
```

Reply:
`WP_Syntex\Polylang\Model\Languages::get()` could returned `PLL_Language` or `false` so PHPStan grumbles

```
  Line   modules\wizard\view-wizard-step-home-page.php
 ------ ------------------------------------------------------------------
  :75    Class PLL_Language referenced with incorrect case: PLL_language.
 ------ ------------------------------------------------------------------
```
even if this line is documented.

Reply:
Reverted in 0803da4d422f7b8b9ac56b0a1d9d14473bddf434